src/module_wrap.cc

```diff
@@ -705,15 +903,38 @@ void ModuleWrap::Resolve(const FunctionCallbackInfo<Value>& args) {
         env, "second argument is not a URL string");
   }
 
-  Maybe<URL> result = node::loader::Resolve(env, specifier_std, url);
-  if (result.IsNothing() || (result.FromJust().flags() & URL_FLAGS_FAILED)) {
-    std::string msg = "Cannot find module " + specifier_std;
-    return node::THROW_ERR_MISSING_MODULE(env, msg.c_str());
+  Maybe<URL> result =
+      node::loader::Resolve(env,
+                            specifier_std,
+                            url);
+  if (result.IsNothing()) {
+    return;
+  }
+
+  URL resolution = result.FromJust();
+  CHECK(!(resolution.flags() & URL_FLAGS_FAILED));
+
+  args.GetReturnValue().Set(resolution.ToObject(env).ToLocalChecked());
```

Comment:
As mentioned before, using `.ToLocalChecked()` is invalid here and can crash the process.

```suggestion
  args.GetReturnValue().Set(resolution.ToObject(env).FromMaybe(Local<Value>()));
```

Or (this would be my personal preference):

```suggestion
  Local<Value> resolution_obj;
  if (resolution.ToObject(env).ToLocal(&resolution_obj))
    args.GetReturnValue().Set(resolution_obj);
```

Reply:
Sure!

Reply:
landed in cc96294216